.travis.yml

```diff
@@ -23,10 +23,5 @@ node_js:
     - '4.2'
     - '5.0'
 
-before_script:
-    - npm install -g jshint
-    - npm install -g jscs
-
 script:
-    - npm run -s lint
```

Comment:
So @davidvgalbraith, did you get upset with jscs/lint yesterday? Or is there some logic behind removing this stuff in this PR?